aiohttp/multipart.py

```diff
@@ -854,6 +856,8 @@ def _apply_content_transfer_encoding(self, stream):
         elif encoding == 'quoted-printable':
             for chunk in stream:
                 yield binascii.b2a_qp(chunk)
+        elif encoding == 'binary':
+            yield from stream
```

Comment:
Have you tried to make actual POST request? It is exception here https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/client_reqrep.py#L427  (got: <class 'generator'> seems pretty clear that is not the same return